plugins/arDominionB5Plugin/modules/jobs/templates/reportSuccess.php

```diff
@@ -4,73 +4,53 @@
-      <?php if (QubitTerm::JOB_STATUS_COMPLETED_ID == $job->statusId) { ?>
-        <i class="fa fa-check-square" id="job-check-color"></i>
-      <?php } elseif (QubitTerm::JOB_STATUS_ERROR_ID == $job->statusId) { ?>
-        <i class="fa fa-exclamation-triangle" id="job-warning-color"></i>
-      <?php } elseif (QubitTerm::JOB_STATUS_IN_PROGRESS_ID == $job->statusId) { ?>
-        <i class="fa fa-cogs" id="job-cogs-color"></i>
-      <?php } ?>
-    </div>
-  </div>
+  <?php
+      $icon = '';
+      if (QubitTerm::JOB_STATUS_COMPLETED_ID == $job->statusId) {
+          $icon = '<i aria-hidden="true" class="fa fa-check-square ms-2 text-success" id="job-check-color"></i>';
+      } elseif (QubitTerm::JOB_STATUS_ERROR_ID == $job->statusId) {
+          $icon = '<i aria-hidden="true" class="fa fa-exclamation-triangle ms-2 text-danger" id="job-warning-color"></i>';
+      } elseif (QubitTerm::JOB_STATUS_IN_PROGRESS_ID == $job->statusId) {
+          $icon = '<i aria-hidden="true" class="fa fa-cogs ms-2 text-secondary" id="job-cogs-color"></i>';
+      }
+      $status = render_value_inline($job->getStatusString()).$icon;
+      echo render_show(__('Status'), $status);
+  ?>
 
-  <div class="job-report-field">
-    <div><?php echo __('Creator'); ?></div>
-    <div><?php echo render_value(QubitJob::getUserString($job)); ?></div>
-  </div>
+  <?php echo render_show(__('Creator'), render_value_inline(QubitJob::getUserString($job))); ?>
 
   <?php if ($job->getObjectModule() && $job->getObjectSlug()) { ?>
-    <div class="job-report-field">
-      <div><?php echo __('Associated record'); ?></div>
-      <div><?php echo link_to(__('Link'), ['module' => $job->getObjectModule(), 'slug' => $job->getObjectSlug()]); ?></div>
-    </div>
+    <?php echo render_show(__('Associated record'), link_to(__('Link'), ['module' => $job->getObjectModule(), 'slug' => $job->getObjectSlug()])); ?>
   <?php } ?>
 
   <?php if ($job->downloadPath) { ?>
-    <div class="job-report-field">
-      <div><?php echo __('Download path'); ?></div>
-      <div><?php echo link_to(__('Link'), public_path($job->downloadPath)); ?></div>
-    </div>
+    <?php echo render_show(__('Download path'), link_to(__('Link'), public_path($job->downloadPath))); ?>
   <?php } ?>
 </section>
 
-<section id="log-area">
+<section id="log-area" class="border-bottom">
   <?php if (!empty($errorOutput)) { ?>
-    <h2><?php echo __('Error(s)'); ?></h2>
-    <div>
-      <pre id="job-log-error-output"><?php echo render_value($errorOutput); ?></pre>
+    <div class="border-bottom">
+      <?php echo render_b5_section_label(__('Error(s)')); ?>
+      <div class="alert alert-danger m-2" role="alert">
+        <pre id="job-log-error-output"><?php echo render_value_inline($errorOutput); ?></pre>
+      </div>
     </div>
   <?php } ?>
 
-  <h2><?php echo __('Log'); ?></h2>
-  <div>
+  <?php echo render_b5_section_label(__('Log')); ?>
+  <div class="bg-secondary alert text-white rounded p-3 m-2">
```

Comment:
Now that I see the `pre` section bellow ... could we remove the `alert` class here and add `pb-0` to the `pre`? I'd prefer to not depend on that temporary CSS if we can.

Reply:
`mb-0` :wink: